packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -56,6 +78,13 @@ export class BucketDeployment extends cdk.Construct {
 
     source.bucket.grantRead(handler);
     props.destinationBucket.grantReadWrite(handler);
+    if (props.distribution) {
+      handler.addToRolePolicy(new PolicyStatement({
+        effect: Effect.ALLOW,
+        actions: ['cloudfront:GetInvalidation', 'cloudfront:CreateInvalidation'],
+        resources: ['*'],
```

Comment:
Why not restrict to this distribution ID?

Reply:
I will re-read the documentation but as I understand it there are no resource-level constraints for invalidations.